### PR TITLE
Set up circleCi building

### PR DESCRIPTION
**Pull request by OFFLlNE**
bitmoji

### CODE REVIEW

File: .circleci/circle.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          at: ~/currency-flags
+      - run:
+          name: Setup npm token
+          command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN_PUBLISH" >> ~/.npmrc
+      - run:
+          name: Build
+          command: npm run build
+      - run:
+          name: Release to GitHub
+          command: npm run release-to-github
```
Comment (oliverviljamaa) [inline .circleci/circle.yml]: This doesn't exist yet, does it?

Comment (OFFLlNE) [reply .circleci/circle.yml]: Seems like it does not exist in the package indeed

File: README.md [hunk 2]
```diff
@@ -21,12 +21,12 @@ This package outputs two CSS files (minified and unminified) with classes for cu
...
-1. Run `npm install`
-1. Add correctly sized (`48x32`) `png` files to `src/flags`
-1. Run `npm run build`
-1. Commit to a new branch, include css files changed by build process
-1. Open a pull request
-1. Draft a release on GitHub with the same version as specified in `package.json`
+## Contributing
+
+1.  Clone repo
+1.  Run `npm install`
```
Comment (oliverviljamaa) [inline README.md]: I don't think you need to do that to add a flag 🙂

Comment (OFFLlNE) [reply README.md]: weelll if you think so.. does not hurt 🙈 changing 😃

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# v1.7.1
+## Fix SCR flag name
+
+The scr.png flag had a typo so it was not showing correctly
+
+# v1.7.0
+## Add ALL AMD AOA and BYN flags
+
+![image](https://user-images.githubusercontent.com/1814752/41033416-5e305a36-697e-11e8-8590-04978e8baef3.png)
+
```
Comment (oliverviljamaa) [inline CHANGELOG.md]: Unnecessary empty line.

**Pull request merged**